aspnetcore/signalr/version-differences.md

```diff
@@ -32,6 +32,10 @@ Automatic reconnects aren't supported in ASP.NET Core SignalR. If the client is
 
 ASP.NET Core SignalR supports JSON, as well as a new binary protocol based on [MessagePack](xref:signalr/messagepackhubprotocol). Additionally, custom protocols can be created.
 
+### Transports
+
+Forever Frame is not supported in ASP.NET Core SignalR.
```

Comment:
Maybe a note on that fact that websockets is much more widely adopted now so we can safely do this. Something along those lines.

Reply:
I'm not sure if we should add that. We don't explain any of our decisions for other differences.